packages/backend/src/services/OrganizationService/OrganizationService.ts

```diff
@@ -135,6 +135,10 @@ export class OrganizationService {
             ),
         );
 
+        // If there are no projects to list after doing permission filtering, we return 404
+        if (filteredProjects.length === 0)
+            throw new NotExistsError('No project exists');
```

Comment:
we return 404 if there are no projects, we should do the same if there are no projects because of permissions, for consistency